### PR TITLE
Don't add already queued items from Diaspora to the queue again

### PR DESCRIPTION
**Pull request by annando**
Already queued items to Diaspora had been added to the queue again and again.

Additionally we now have the item guid in the queue to improve further checks.

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -855,6 +855,7 @@ CREATE TABLE IF NOT EXISTS `queue` (
 	`id` int NOT NULL auto_increment COMMENT '',
 	`cid` int NOT NULL DEFAULT 0 COMMENT '',
 	`network` varchar(32) NOT NULL DEFAULT '' COMMENT '',
+	`guid` varchar(255) NOT NULL DEFAULT '' COMMENT '',
```
Comment (tobiasd) [inline database.sql]: Could you add it to the DB structure docs as well?

Comment (annando) [reply database.sql]: Eventually the documentation will be created from the database structure, that's why there is now this comment field. So I would postpone the manual editing until that time. (That should be in this release as well).

Comment (annando) [reply database.sql]: See issue https://github.com/friendica/friendica/issues/4419 and https://github.com/friendica/friendica/issues/4420

Comment (rabuzarus) [reply database.sql]: But it would be nice if you could insert something in the new comment field and don't leave it empty ;-)

Comment (annando) [reply database.sql]: Documentation done. :-P

Comment (MrPetovan) [reply database.sql]: I can’t see the field comment, did you forget to commit/push it?

Comment (annando) [reply database.sql]: Note to myself: Commit before push, not the reverse ...

**Pull request merged**